dscreator/sources/base.py

```diff
@@ -49,6 +55,11 @@ class NamedTrajectory(FeatureBase):
     datetime_list: List[datetime]
     locations: List[Point]
 
+    def __add__(self, nt):
+        return NamedTrajectory(datetime_list=self.datetime_list + nt.datetime_list,
+                               locations = self.locations + nt.locations,
+                               array_list= FeatureBase(self.array_list).__add__(nt.array_list))
```

Comment:
This looks quite strange with creating a new featurebase just for calling the dunder that returns the array